lightningd/jsonrpc.c

```diff
@@ -317,6 +317,7 @@ struct json_result *null_response(const tal_t *ctx)
 	return response;
 }
 
+
```

Comment:
Unnecessary change?

Reply:
was from the github editor, remove it at next commit :-)